Source/JavaScriptCore/inspector/agents/InspectorDebuggerAgent.h

```diff
@@ -252,8 +253,28 @@ class JS_EXPORT_PRIVATE InspectorDebuggerAgent
     InjectedScriptManager& m_injectedScriptManager;
     UncheckedKeyHashMap<JSC::SourceID, JSC::Debugger::Script> m_scripts;
 
-    using BlackboxParameters = std::tuple<String /* url */, bool /* caseSensitive */, bool /* isRegex */>;
-    UncheckedKeyHashMap<BlackboxParameters, UncheckedKeyHashSet<JSC::Debugger::BlackboxRange>> m_blackboxedURLs;
+    struct BlackboxedScript {
+        String url;
+        bool caseSensitive { false }; // FIXME: should this be `true`?
```

Comment:
Yes

Reply:
<https://github.com/WebKit/WebKit/pull/43160> (_Web Inspector: Debugger: blackboxing should default to case sensitive_)